tokens/package-lock.yml

```diff
@@ -0,0 +1,8 @@
+packages:
```

Comment:
Should this be here, any context?

Reply:
this file auto-generates when you run `dbt clean`, `dbt deps`, `dbt compile`
since tokens is a new project, we didn't have on in there
you will notice the main spellbook project has the same file:
https://github.com/duneanalytics/spellbook/blob/main/package-lock.yml

Reply:
first few commits, i manually removed the file, but it kept coming back. i ended up including in final.